### ISSUE TITLE
Issue when parsing some permafrost data

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/data-api#31]
http://earthmaps.io/permafrost/point/62.6541/-151.5155

Error stack:
```
File "data-api/routes/permafrost.py", line 84, in package_obu_magt
    temp = round(obu_magt_resp["features"][0]["properties"]["GRAY_INDEX"], 2)
TypeError: type NoneType doesn't define __round__ method
```

**Comment by charparr** [comment]
Looks like this particular error is caused by trying to round data when there is one. I think adding a function to catch `NoneType` just as we do for `-9999` and the like to `validate_data` module is the move here.

**Comment by charparr** [comment]
Closed by #74
